apps/user-office-backend/src/datasources/postgres/records.ts

```diff
@@ -404,6 +404,8 @@ export interface SEPRecord {
   readonly code: string;
   readonly description: string;
   readonly number_ratings_required: number;
+  readonly grade_guide: string;
+  readonly custom_grade_guide: boolean;
```

Comment:
It looks good but does `customGradeGuide` need to be `boolean | null`?

Reply:
It gives a type error with that change where `null` is not assignable to `boolean`.
After some digging around I found that because we have `strictNullChecks` enabled in the config `null` and `undefined` will have their own distinct types and an error can be expected if theyre used where concrete value is expected and as the `custom_grade_guide` here is a checkbox it can only be true or false and can't be left unchecked.
Hope that makes more sense 🙂

Reply:
Is that error because the SEP model would also need updating to `boolean | null`? My line of thinking was that it doesn't look like we are modifying any of the existing SEPs to set the `customGradeGuide` to false, so those rows will have a null value, and I wasn't sure how reading those database rows will be handled if we expect a boolean.